client/post-editor/editor-categories-tags/accordion.jsx

```diff
@@ -1,88 +1,85 @@
+import { connect } from 'react-redux';
 import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+import { get, toArray } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import Accordion from 'components/accordion';
 import AccordionSection from 'components/accordion/section';
+import EditorDrawerLabel from 'post-editor/editor-drawer/label';
 import Gridicon from 'components/gridicon';
-import Categories from 'post-editor/editor-categories';
+import TermSelector from 'post-editor/editor-term-selector';
 import Tags from 'post-editor/editor-tags';
-import TermStore from 'lib/terms/store';
-import siteUtils from 'lib/site/utils';
 import InfoPopover from 'components/info-popover';
-import { isPage } from 'lib/posts/utils';
 import unescapeString from 'lodash/unescape';
-
-module.exports = React.createClass( {
-	displayName: 'EditorCategoriesTagsAccordion',
-
-	propTypes: {
-		site: React.PropTypes.object,
-		post: React.PropTypes.object,
-		// passed down from CategoryListData
-		categories: React.PropTypes.array,
-		categoriesFound: React.PropTypes.number,
-		categoriesHasNextPage: React.PropTypes.bool,
-		categoriesFetchingNextPage: React.PropTypes.bool,
+import Notice from 'components/notice';
+import { addSiteFragment } from 'lib/route';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getEditorPostId } from 'state/ui/editor/selectors';
+import { getEditedPostValue } from 'state/posts/selectors';
+import { getSiteOption, isJetpackMinimumVersion, getSiteSlug } from 'state/sites/selectors';
+import { getTerm } from 'state/terms/selectors';
+
+export class EditorCategoriesTagsAccordion extends Component {
+	static propTypes = {
+		site: PropTypes.object,
+		post: PropTypes.object,
+		translate: PropTypes.func,
+		postTerms: PropTypes.object,
+		postType: PropTypes.string,
+		defaultCategory: PropTypes.object,
+		isTermsSupported: PropTypes.bool,
+		siteSlug: PropTypes.string,
 		// passed down from TagListData
-		tags: React.PropTypes.array,
-		tagsHasNextPage: React.PropTypes.bool,
-		tagsFetchingNextPage: React.PropTypes.bool
-	},
-
-	isCategoriesLoading: function() {
-		if ( ! this.props.site ) {
-			return true;
-		}
-
-		return this.props.categoriesFetchingNextPage;
-	},
-
-	getCategories: function() {
-		var defaultCategory;
-
-		if ( ! this.props.post ) {
-			return [];
-		}
+		tags: PropTypes.array,
+		tagsHasNextPage: PropTypes.bool,
+		tagsFetchingNextPage: PropTypes.bool
+	};
 
-		if ( this.props.post.category_ids ) {
-			return this.props.post.category_ids;
-		}
-
-		defaultCategory = siteUtils.getDefaultCategory( this.props.site );
-		if ( defaultCategory ) {
-			return [ defaultCategory ];
-		}
-
-		return [];
-	},
+	renderJetpackNotice() {
+		const { translate, siteSlug } = this.props;
+		return (
+			<Notice status="is-warning" showDismiss={ false }>
+				{ translate( 'You must update Jetpack to use this feature. {{update}}Update Now{{/update}}', {
```

Comment:
Did you try to use `<NoticeAction />` instead of the inline link? Not sure if that adjusts well to small width elements like it does for smaller viewports.